docs/Makefile

```diff
@@ -1,9 +1,9 @@
 GOMD2MAN = go-md2man
 
-docs: $(patsubst %.md,%.1,$(wildcard *.md)) container-storage.conf.5
+docs: $(patsubst %.md,%.1,$(wildcard *.md)) containers-storage.conf.5
```

Comment:
The wildcard still has one issue -> `containers-storage.conf.5.1`. Would you fix that as well?

Reply:
Sure